packages/core/src/scope_arena.rs

```diff
@@ -144,6 +83,12 @@ impl VirtualDom {
             id: scope.id,
         });
 
+        if scope.suspended.get() {
```

Comment:
Should this only run when the rendered value is none?

Reply:
Good point, yes